src/core/handle.ml

```diff
@@ -133,6 +134,89 @@ let handle_require_as : popt -> sig_state -> Path.t -> ident -> sig_state =
   let path_map = PathMap.add p id.elt ss.path_map in
   {ss with aliases; path_map}
 
+(** Symbol properties needed for the signature are packed together *)
+type sig_symbol =
+  { expo   : expo        ;
+    prop   : prop        ;
+    mstrat : match_strat ;
+    x      : ident       ;
+    a      : term        ;
+    impl   : bool list   ;
+    d      : term option ; }
```

Comment:
x, a and d are names that are too short. Thanks to replace for instance by ident, typ and def.

Reply:
fixed in 7e37bcb